src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.Compilation.targets

```diff
@@ -111,12 +111,6 @@ Copyright (c) .NET Foundation. All rights reserved.
       <UseSharedCompilation>true</UseSharedCompilation>
     </PropertyGroup>
 
-    <PropertyGroup>
```

Comment:
This one makes me slightly nervous. In the general case, this target is invoked after `CoreCompile` and incremental builds will evaluate property and item group, however there might be code-paths where this target is invoked without `CoreCompile` being evaluated. At the same time, I'd much rather keep have this evaluation in the Razor targets.

Should we reach out to the Roslyn \ CoreSDK folks and see if there's a way they could evaluate this in the global context instead?

Reply:
Ya this makes me nervous too. Sorry, not sure I quite followed your first explanation why remove this at all?

Reply:
Ahh, for some reason I thought you wanted me to remove this as part of https://github.com/aspnet/AspNetCore-Tooling/pull/405#issuecomment-481472031, when all it asked was for it to be updated.

Reply:
done